### DISCUSSION TITLE
How to set restriction to 2 or more songs

### DISCUSSION THREAD

**Discussion by enchanteddaemon** [Lost-MSth/Arcaea-server#113] [Q&A]
![Screenshot_2023-05-21-21-20-36-87_9e8df3d0c7c1f50248b6ee043a653d26.jpg](https://github.com/Lost-MSth/Arcaea-server/assets/100534753/ae1c1edb-31b2-4563-9161-351fa5df4e83)

i can only do 1 in every tile

**Comment by Lost-MSth** [comment]
`restrict_ids: list[str]`

**Reply by enchanteddaemon** [reply]
how about to restrict only on byd

**Reply by Lost-MSth** [reply]
`restrict_difficulty: int`
